### ISSUE TITLE
feature request: postAsJson

### ISSUE THREAD

**Issue by iamstarkov** [nordnet/nordnet-next-api#4]
i would like to add `postAsJson` or similar which will hide `{ 'Content-type': 'application/json' })` verbosity:

``` js
// verbose
function login(username, password) {
  const jsonType = { 'Content-type': 'application/json' };
  const payload = { username, password };
  return api.post('/api/login', payload, jsonType);
}

// new and shiny
const login = (username, password) => api.post('/api/login', { username, password });
```

what do you think?

**Comment by dimchez** [comment]
@iamstarkov I think `postJSON` would be a good shorthand method to have. Feel free to send a PR :)

**Comment by dimchez** [comment]
Closed by e445e20eb9ec72e611cf9d5120e93d2ddbc437cc
